utils/notification_service.py

```diff
@@ -741,15 +741,15 @@ def add_path(self, path: str, gpu: str = None):
     ci_title = os.environ.get("CI_TITLE")
     ci_url = os.environ.get("CI_COMMIT_URL")
 
-    commit_number = ci_url.split("/")[-1]
-    ci_detail_url = f"https://api.github.com/repos/huggingface/transformers/commits/{commit_number}"
-    ci_details = requests.get(ci_detail_url).json()
-    ci_author = ci_details["author"]["login"]
-
     if ci_title is not None:
         assert ci_url is not None
         ci_title = ci_title.strip().split("\n")[0].strip()
 
+        commit_number = ci_url.split("/")[-1]
```

Comment:
Suggestion to add a comment for explainability

```suggestion
        # Retrieve the PR title and author login to complete the report 
        commit_number = ci_url.split("/")[-1]
```